README.md

```diff
@@ -1,69 +1,91 @@
+
+## Prerequisites
+
+* LocalStack Pro with the [`localstack` CLI](https://docs.localstack.cloud/getting-started/installation/#localstack-cli).
+* [Serverless Application Model](https://docs.localstack.cloud/user-guide/integrations/aws-sam/) with the [samlocal](https://github.com/localstack/aws-sam-cli-local) installed.
+* [Python 3.9 installed](https://www.python.org/downloads/).
+* [Node.js](https://nodejs.org/en/download) with npm package manager.
+* [Artillery](https://artillery.io/docs/guides/overview/welcome.html) to generate some load towards both the apis.
+* [`yq`](https://github.com/mikefarah/yq#install) and[`jq`](https://jqlang.github.io/jq/download/) for running the deployment script. 
+
+Start LocalStack Pro with the `LOCALSTACK_API_KEY` pre-configured:
+
+```shell
+export LOCALSTACK_API_KEY=<your-api-key>
+localstack start
+```
 
-## Deploy the sample application
+> If you prefer running LocalStack in detached mode, you can add the `-d` flag to the `localstack start` command, and use Docker Desktop to view the logs.
 
-The Serverless Application Model Command Line Interface (SAM CLI) is an extension of the AWS CLI that adds functionality for building and testing Lambda applications. 
-It uses Docker to run your functions in an Amazon Linux environment that matches Lambda. It can also emulate your application's build environment and API.
+## Instructions
 
-To use the SAM CLI, you need the following tools.
+You can build and deploy the sample application on LocalStack by running our `Makefile` commands. Run `make deploy` to create the infrastructure on LocalStack. Run `make stop` to delete the infrastructure by stopping LocalStack.
 
-* SAM CLI - [Install the SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html)
-* [Python 3.9 installed](https://www.python.org/downloads/)
-* Docker - [Install Docker community edition](https://hub.docker.com/search/?type=edition&offering=community)
+Alternatively, here are instructions to deploy it manually step-by-step.
 
-## Deploy RDS Aurora Cluster with RDS Proxy
+### Deploy RDS Aurora Cluster with RDS Proxy
 
 **Note:** If you have already provisioned RDS Aurora cluster with RDS Proxy, you can skip 
 this step and follow [these steps](#deploy-serverless-workload-using-rds-aurora-as-backend) instead.
 
-This stack will take care of provisioning RDS Aurora Mysql along with RDS proxy fronting it inside
+This stack will take care of provisioning RDS Aurora PostgreSQL along with RDS proxy fronting it inside
 a VPC with 3 private subnet. Required parameters needed by [next step](#deploy-serverless-workload-using-rds-aurora-as-backend)
 is also provided as stack output.
 
 ```bash
-    sam build -t rds-with-proxy.yaml --use-container
-    sam deploy -t rds-with-proxy.yaml --guided
+    samlocal build -t rds-with-proxy.yaml --use-container
+    samlocal deploy -t rds-with-proxy.yaml --guided
 ```
```

Comment:
Q: do we also need to create the user in that step, e.g. by running `create-user.py`?